collectors/persistentvolume.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collectors
+
+import (
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"golang.org/x/net/context"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/tools/cache"
+)
+
+var (
+	descPersistentVolumeStatusPhase = prometheus.NewDesc(
+		"kube_persistentvolume_status_phase",
+		"The phase indicates if a volume is available, bound to a claim, or released by a claim.",
+		[]string{
+			"namespace",
+			"persistentvolume",
```

Comment:
after looking at `kube_pod_status_phase`, seems using `persistentvolume` is fine. Sorry for the inconvenience.